appveyor.yml

```diff
@@ -40,6 +40,13 @@ cache:
   - .cargo -> rust-toolchain
   - .ccache
 
+install:
+  - appveyor-retry appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init.exe -y --default-host x86_64-pc-windows-msvc --default-toolchain none
+  - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
+  - rustc -V
+  - cargo -V
```

Comment:
Should we modify the cache instructions above this block?

Reply:
It looks like this caches the `.servo` directory including the extracted rustc and cargo downloaded by mach. Do we want to cache toolchains downloaded by rustup?

Reply:
Yes, since it makes builds a bit faster.

Reply:
Alright, done.